swap_meet/item.py

```diff
@@ -1,2 +1,17 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        self.id = id if id is not None else uuid.uuid4().int
+        self.condition = condition
+
+    def get_category(self):
+        return self.__class__.__name__
+    
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+
+    conditions = {0: "New", 1: "Mint", 2: "Good", 3: "Fair", 4: "Heavily used", 5: "Damaged"}
```

Comment:
I would opt to move this inside of the `condition_description` method since it is the only function that will use it. This could also be a time to use match case! 
```python
    def condition_description(self):
        match self.condition:
            case 0:
                return "Throw Away"
            case 1:
                return "Eh"
            case 2:
                return "Better..."
            case 3:
                return "..."
            case 4: 
                return "Okay?!"
            case 5:
                return "Yes!"
```